src/Sleep.js

```diff
@@ -0,0 +1,12 @@
+class Sleep{
+  constructor(data, id) {
```

Comment:
Since we were filtering by idea for our sleep class, I took out the filter by id in the User methods to DRY it up a bit

Reply:
That sounds like a great idea. I don't think we need the ID here.